docs/dev/_index.md

```diff
@@ -492,8 +492,6 @@ for i := 0; i < len(detections); i++ {
 }
 ```
 
-Be sure to import `"go.viam.com/rdk/utils"` at the beginning of your file.
```

Comment:
[feel free to ignore] I assume this is something you caught incidentally and included but just flagging since it's a big PR to make sure this is meant to be deleted here

Reply:
I think Sierra added this accidentally - but yeah it's meant to get deleted